contracts/StorageManager.sol

```diff
@@ -17,13 +20,13 @@ contract StorageManager {
     Offer represents:
      - utilizedCapacity: how much is capacity is utilized in Offer.
      - totalCapacity: total amount of bytes offered.
-     - billingPlans: maps a billing period to a billing price. When a price is 0, the period is not offered.
+     - billingPlansForToken: maps a whitelisted token to billing period to a billing price. When a price is 0, the period is not offered. Address 0 stands for the native currency. By convention, the 0-address stands for the native currency
```

Comment:
Comment seems quite redundant the last 2 sentences I would leave only one.

Reply:
Correct. Will remove. Thx

Reply:
Done